pkg/storage/util/util.go

```diff
@@ -0,0 +1,65 @@
+package util
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/operator-framework/operator-sdk/pkg/sdk"
+
+	coreapi "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metaapi "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/util/retry"
+
+	regopapi "github.com/openshift/cluster-image-registry-operator/pkg/apis/imageregistry/v1alpha1"
+	"github.com/openshift/cluster-image-registry-operator/pkg/clusterconfig"
+)
+
+func GetConfig(name string, namespace string) (*clusterconfig.Config, error) {
+	cfg, err := clusterconfig.Get()
```

Comment:
this needs to be getting this from the install-config and cluster-wide credentials.

And this function is poorly named for something that has side-effects of setting up a secret.  Refactor it into a helper to setup the credential secret, and (if still appropriate) a helper to get the config.

Reply:
and clusterconfig.Get() of course needs to be updated to just get the config directly from the install-config and get rid of the intermediary configmap.

Reply:
updated in next push